src/components/Patient/EncounterQuestionnaire.tsx

```diff
@@ -43,7 +43,9 @@ export default function EncounterQuestionnaire({
                 navigate(`/patient/${patientId}/updates`);
               }
             }}
-            onCancel={() => goBack()}
+            onCancel={() =>
```

Comment:
Let's not add a fallback for this one, since you can travel to Questionnaire via patient or encounter, and a user may have access to patient but not to the encounter.